### PR TITLE
WD-9413 - add new /identity page

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Add /identity page and reference it in /diversity page

## QA

- [demo link](https://canonical-com-1213.demos.haus/careers/company-culture/identity)
- [copy doc](https://docs.google.com/document/d/1_-EhhObUF8vl95CypcJEpFcMdvDU4N8ak9kCLvZMtyw/edit)
- [figma](https://www.figma.com/file/RM6wQKCy3u7AyvNJAGQ70v/canonical.com%2Fcareers%2Fdiversity%2Fidentity?node-id=17%3A562&mode=dev)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is done correctly according to figma and copy doc
- Check that the page is referenced on [/diversity](https://canonical-com-1213.demos.haus/careers/company-culture/identity)

## Issue / Card
[WD-9413](https://warthogs.atlassian.net/browse/WD-9413)

[WD-9413]: https://warthogs.atlassian.net/browse/WD-9413?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: static/sass/styles.scss [hunk 1]
```diff
@@ -846,3 +846,9 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
   background-color: $colors--dark-theme--background-default;
   color: $colors--dark-theme--text-default;
 }
+
+.p-highlight--vertical {
+  margin-left: 0;
+  padding-left: 1rem;
+  @include vf-highlight-bar(#E95420, left);
+}
```
Comment (mtruj013) [inline static/sass/styles.scss:854]:
```suggestion
.p-highlight--vertical {
  @include vf-highlight-bar(#E95420, left);
  
  margin-left: 0;
  padding-left: 1rem;
}
```
We usually put these at the top

File: templates/careers/company-culture/identity.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+<section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule"/>
+      <div class="col">
+        <h1 class="p-heading--2">Proactive appointments</h1>
+      </div>
+      <div class="col">
+        <p>We must attract the best colleagues from around the world.</p>
+        <p>‘Best’ is always a complex calculation that weighs many factors. People are not mechanical - they have strengths and weaknesses, and we weigh those up in making hiring decisions. Do we hire a developer with more experience in the language we use? Or a better track record of documenting their work? Or more familiarity with open source and community? Or a better track record of learning quickly? We often have candidates who are the best in one of those elements but not all of them. The ‘best’ candidate is usually the one who will round out and strengthen the team.</p>
```
Comment (carkod) [inline templates/careers/company-culture/identity.html:60]: People are not mechanical &mdash; they have...

It's not the standard dash

File: templates/careers/company-culture/identity.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <p>We have no demographic quotas to fill, and every single appointment will be made individually on merit. In order to attract talent from every group, we strive to reflect the best of every group in our company.</p>
+      </div>
+    </div>
+</section>
+
+<section class="p-section--deep">
+    <div class="row--50-50">
+      <hr class="p-rule"/>
+      <div class="col">
+        <h1 class="p-heading--2">On language at work</h1>
```
Comment (mtruj013) [inline templates/careers/company-culture/identity.html:72]:
```suggestion
        <h2>On language at work</h2>
```

File: templates/careers/company-culture/identity.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        <p>Many candidates who were previously rejected based on their resumes applying for a team, do very well in this assessment process and ultimately get strong positive feedback from that team during interviews.</p>
+      </div>
+    </div>
+</section>
+
+<section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule"/>
+      <div class="col">
+        <h1 class="p-heading--2">Proactive appointments</h1>
```
Comment (mtruj013) [inline templates/careers/company-culture/identity.html:56]:
```suggestion
        <h2>Proactive appointments</h2>
```

File: templates/careers/company-culture/identity.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule"/>
+      <div class="col">
+        <h1 class="p-heading--2">Impact on our hiring process</h1>
```
Comment (mtruj013) [inline templates/careers/company-culture/identity.html:40]:
```suggestion
        <h2>Impact on our hiring process</h2>
```

File: templates/careers/company-culture/identity.html [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+            <p>We gain strength from diversity, and actively seek participation from those who enhance it. This code of conduct exists to ensure that diverse groups collaborate to mutual advantage and enjoyment. We will challenge prejudice that could jeopardise the participation of any person in the project.</p>
+          </div>
+        </div>
+        <p>This statement is as true of Canonical as it is of the Ubuntu project.</p>
+        <p>We did not set out to create the world’s most profitable software company at any cost. We set out to bring the benefits of open source to all humanity by making it easier and cheaper to consume and easier to improve, to level the playing field for innovators, entrepreneurs, students and digital consumers all across the globe.</p>
+        <p>We also set out to create a productive work environment for those who want to contribute to the open source revolution we are creating - living and working wherever they prefer, regardless of their identity. We continue to seek ways to improve on these fronts.</p>
```
Comment (mtruj013) [inline templates/careers/company-culture/identity.html:30]:
```suggestion
        <p>We also set out to create a productive work environment for those who want to contribute to the open source revolution we are creating &ndash; living and working wherever they prefer, regardless of their identity. We continue to seek ways to improve on these fronts.</p>
```
For the ones below as well

**Review by mtruj013 — APPROVED** [review]
Thanks @lizzochek , just some super minor comments

File: templates/careers/company-culture/identity.html (hunk 2 above)
Comment (mtruj013) [reply templates/careers/company-culture/identity.html:60]: you beat me to it! Didn't know you were taking a look, my bad :)

**Pull request merged**